evap/staff/tests/test_views.py

```diff
@@ -872,6 +878,11 @@ def test_course_create(self):
         self.assertEqual(Course.objects.get().name_de, "lfo9e7bmxp1xi")
 
 
+@override_settings(REWARD_POINTS=[
+    (1.0 / 3.0, 1),
+    (2.0 / 3.0, 2),
+    (3.0 / 3.0, 3),
```

Comment:
just 1 / 3, without the decimals. integer divisions result in floats in python 3. feel free to change that in the settings and in rewards/tests/test_tools.py as well, but you don't have to.